src/clients/http_client/http_client.ts

```diff
@@ -180,7 +180,13 @@ class HttpClient {
   ): Promise<RequestReturn> {
     return fetch(url, options)
       .then(async (response: Response) => {
-        const body = await response.json();
+        let body;
+
+        try {
+          body = await response.json();
+        } catch {
+          body = {};
+        }
```

Comment:
Here's a suggestion to simplify this :) 

```suggestion
        const body = await response.json().catch(() => ({}));
```

Although now that I'm looking at this file @paulomarg we should refactor this. Thanks @zookcorp!